CoreAssignment_5/Assignment/Deserialisation.java

```diff
@@ -0,0 +1,26 @@
+package Assignment;
+
+import java.io.FileInputStream;
+import java.io.ObjectInputStream;
+import java.util.ArrayList;
+
+public class Deserialisation{
+    public static void main(String[] args){
+        
+        ArrayList<Student> info = new ArrayList<>();
+try{
+            ObjectInputStream ob = new ObjectInputStream(new FileInputStream("D:\\Hotwax\\assignments\\CoreAssignment_5\\Output1.ser"));
```

Comment:
you can use relative path otherwise if you run this on other platform like linux then this will return error